Firmware/temperature.h

```diff
@@ -255,11 +255,11 @@ void check_fans();
 
 void check_min_temp();
 void check_max_temp();
-
+void fanSpeedErrorBeep(const char *serialMsg, const char *lcdMsg);
 
 #endif
 
 extern unsigned long extruder_autofan_last_check;
 extern uint8_t fanSpeedBckp;
 extern bool fan_measuring;
-
+extern bool resume_print;
```

Comment:
document variable usage - it is not completely clear, what it is supposed to do/guard/set. It looks like a flag saying "the print is being resumed right now", but not sure...